environments/IC_bench_4.2.6.yml

```diff
@@ -14,9 +14,12 @@ dependencies:
   - conda-forge::pathlib
   - conda-forge::nbconvert
   - conda-forge::Cython==0.29.36
+  - conda-forge/label/broken::pytorch
```

Comment:
label/broken???

Reply:
Lol anaconda.org said use one of the following 1. conda install conda-forge::pytorch
2. conda install conda-forge/label/broken::pytorch and I impulsively picked the weird one. I switched it to conda install conda-forge::pytorch. Thanks for pointing  this out.

Reply:
Whoops fixed it